### DISCUSSION TITLE
I can't even pay for it

### DISCUSSION THREAD

**Discussion by chainyo** [community/community#19194] [Copilot Conversations] [answered]
Hi everyone! I have had access to Copilot since the first beta day, and I'm ready to pay for it now, but I've been facing a weird problem since yesterday.

No matter what I try (uninstall, reinstall, restart vscode, pay 10 $ or 100 $, choose to keep the free plan), I can't get my access back. The Copilot extension keeps saying that I need to authenticate on Github and activate my access to it, but after like 100 tries and errors, I can't understand why it's going wrong?

If anyone could help me?

PayPal or Credit card fail.
![image](https://user-images.githubusercontent.com/50595514/175104460-c5b75a23-01a0-4fac-99ad-f697a835b3dc.png)
VsCode fails
![image](https://user-images.githubusercontent.com/50595514/175104533-c6c38643-b337-4cd7-9b46-54ac522e060a.png)

**Comment by NeuralAIM** [comment]
#19093 - Just because you were born in this country, you cannot make purchases or use a free test in 60 days!

**Reply by chainyo** [reply]
Sorry for you. I am in France 🇫🇷.

**Comment by ShinChven** [comment, marked as answer]
I had this problem, and solved it by contacting the support.
https://github.com/github-community/community/discussions/19123

Open a ticket here, so support team may help you.
https://support.github.com/tickets/personal/0

**Comment by melvincwng** [comment]
@ChainYo, kindly raise a ticket to [Github Support](https://support.github.com/contact/account) for them to investigate/resolve the issue.

Hope this clarifies.
